### PR TITLE
Isolate scenario load approach: xml vs csv

### PR DESCRIPTION
**Pull request by carloscaldas**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1881)

### CODE REVIEW

File: src/main/scala/beam/utils/scenario/urbansim/ParquetScenarioReader.scala [hunk 1]
```diff
@@ -94,8 +95,19 @@ object ParquetScenarioReader extends UrbanSimScenarioReader with LazyLogging {
     val personId = getIfNotNull(rec, "person_id").toString
     val householdId = getIfNotNull(rec, "household_id").toString
     val age = getIfNotNull(rec, "age").asInstanceOf[Long].toInt
+    val isFemaleValue = {
+      val value = getIfNotNull(rec, "sex").asInstanceOf[String]
```
Comment (REASY) [inline src/main/scala/beam/utils/scenario/urbansim/ParquetScenarioReader.scala]: This will fail in runtime because `sex` is `int64` or `Long` in Java.

Comment (carloscaldas) [reply src/main/scala/beam/utils/scenario/urbansim/ParquetScenarioReader.scala]: fixed

File: src/main/scala/beam/agentsim/agents/choice/logit/MultinomialLogit.scala [hunk 2]
```diff
@@ -2,8 +2,8 @@ package beam.agentsim.agents.choice.logit
 
 import scala.collection.immutable.SortedSet
 import scala.util.Random
-
 import com.typesafe.scalalogging.LazyLogging
+import org.matsim.api.core.v01.population.Person
```
Comment (REASY) [inline src/main/scala/beam/agentsim/agents/choice/logit/MultinomialLogit.scala]: Remove unused namespace/types

File: src/main/scala/beam/agentsim/agents/choice/mode/ModeChoiceMultinomialLogit.scala [hunk 3]
```diff
@@ -11,18 +10,21 @@ import beam.router.model.{EmbodiedBeamLeg, EmbodiedBeamTrip}
 import beam.sim.BeamServices
 import beam.sim.config.BeamConfig.Beam.Agentsim.Agents
 import beam.sim.config.BeamConfig.Beam.Agentsim.Agents.ModalBehaviors
-import beam.sim.population.AttributesOfIndividual
+import beam.sim.population.{AttributesOfIndividual, HouseholdAttributes}
```
Comment (REASY) [inline src/main/scala/beam/agentsim/agents/choice/mode/ModeChoiceMultinomialLogit.scala]: Remove unused namespace/types

File: src/main/scala/beam/agentsim/agents/choice/logit/MultinomialLogit.scala (hunk 2 above)
Comment (REASY) [inline src/main/scala/beam/agentsim/agents/choice/logit/MultinomialLogit.scala]: Remove unused namespace/types

File: src/main/scala/beam/sim/population/PopulationAttributes.scala [hunk 4]
```diff
@@ -12,6 +12,7 @@ import org.matsim.api.core.v01.population._
 import beam.sim.BeamServices
 import beam.agentsim.agents.modalbehaviors.ModeChoiceCalculator._
 import beam.router.RouteHistory.LinkId
+import beam.utils.scenario.{PersonId, PersonInfo}
```
Comment (REASY) [inline src/main/scala/beam/sim/population/PopulationAttributes.scala]: Remove unused namespace/types

**Comment by michaz** [comment]
Merge #1903 into this branch to update it to recent development.

I like this, it's much clearer now that different kinds of population do not appear from different places at different points in time.

**Comment by michaz** [comment]
Can we merge it once UrbansimSpec passes?

**Pull request merged**